src/app/profiles/App/Pages/index.tsx

```diff
@@ -3,18 +3,23 @@ import React from 'react';
 import Profiles from './Profiles';
 import Error from './Error';
 import Subscriptions from './Subscriptions';
-import { CONTRIBUTORS_PATH } from '../../routes';
+import { path } from '../../../routes';
+import useChangeLanguage from '../../../hooks/useChangeLanguage';
 
-const Pages = () => (
-  <>
-    <Switch>
-      <Redirect exact path="/" to={CONTRIBUTORS_PATH} />
-      <Redirect from={'/en/*'} to={CONTRIBUTORS_PATH} />
-      <Route path={CONTRIBUTORS_PATH} component={Profiles} />
-      <Route path="/mes-abonnements" component={Subscriptions} />
-      <Route component={Error} />
-    </Switch>
-  </>
-);
+const Pages = () => {
+  const lang = useChangeLanguage();
+  return (
+    <>
+      <Switch>
+        <Redirect exact path="/" to={path.fr.CONTRIBUTORS} />
+        <Redirect exact path="/en" to={path.en.CONTRIBUTORS} />
+        <Route path={path.fr.CONTRIBUTORS} component={Profiles} />
+        <Route path={path.en.CONTRIBUTORS} component={Profiles} />
+        <Route path={path[lang].SUBSCRIPTIONS} component={Subscriptions} />
+        <Route component={Error} />
+      </Switch>
```

Comment:
I'm not saying route config is the way to go but it might be interesting if it's getting too complex otherwise:
> https://reactrouter.com/web/example/route-config